examples/ex18p.cpp

```diff
@@ -107,23 +108,56 @@ int main(int argc, char *argv[])
                   "Enable or disable GLVis visualization.");
    args.AddOption(&vis_steps, "-vs", "--visualization-steps",
                   "Visualize every n-th timestep.");
+   args.ParseCheck();
 
-   args.Parse();
-   if (!args.Good())
+   // 2. Read the mesh from the given mesh file.
+   Mesh mesh = mesh_file.empty() ? EulerMesh(problem) : Mesh(mesh_file);
+   const int dim = mesh.Dimension();
+   const int num_equations = dim + 2;
+
+   if (problem == 5)
    {
-      if (Mpi::Root()) { args.PrintUsage(cout); }
-      return 1;
+      mesh.Transform([](const Vector &x, Vector &y)
+      {
+         y = x;
+         y *= 0.5;
+      });
+   }
+   //    Refine the mesh to increase the resolution. In this example we do
+   //    'seq_ref_levels' of uniform refinement, where 'seq_ref_levels' is a
+   //    command-line parameter.
```

Comment:
```suggestion
   // Refine the mesh to increase the resolution. In this example we do
   // 'seq_ref_levels' of uniform refinement, where 'seq_ref_levels' is a
   // command-line parameter.
```
Very minor formatting